internal/repository/sync.go

```diff
@@ -64,8 +68,49 @@ func Clone(ctx context.Context, remoteURL, dir, initialBranch string) (*Reposito
 		return nil, errors.Join(ErrCloningRepository, err)
 	}
 
+	sort.Slice(expectedRootKeys, func(i, j int) bool {
+		return expectedRootKeys[i].KeyID < expectedRootKeys[j].KeyID
+	})
+
 	repository := &Repository{r: r}
 
 	slog.Debug("Verifying HEAD...")
+	if len(expectedRootKeys) > 0 {
+		slog.Debug("Getting root keys...")
+		state, err := policy.LoadCurrentState(ctx, r)
```

Comment:
We want to bootstrap the _first_ set of root keys, not the latest. Notice that when a root key changes after the first version of root, gittuf uses the prior version to evaluate the latest set of root keys. The bootstrap issue is about the first one.

Reply:
Fixed